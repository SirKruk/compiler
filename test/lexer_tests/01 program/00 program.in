# sample program
y = 3;
f = () -> 2 * (y + 3) - 1 / 4;
{ "string"; (1, 2); }
