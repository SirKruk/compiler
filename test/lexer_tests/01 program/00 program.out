INPUT_START
ID(y)
EQ
INT(3)
SEMI
ID(f)
EQ
LP
RP
SUB
MORE
INT(2)
MUL
LP
ID(y)
ADD
INT(3)
RP
SUB
INT(1)
DIV
INT(4)
SEMI
LC
STRING(string)
SEMI
LP
INT(1)
COMMA
INT(2)
RP
SEMI
RC
INPUT_STOP
